xknx/devices/climate.py

```diff
@@ -130,7 +131,8 @@ def __init__(self,
             group_address_on_off,
             group_address_on_off_state,
             device_name=self.name,
-            after_update_cb=self.after_update)
+            after_update_cb=self.after_update,
+            invert=on_off_inverted)
```

Comment:
Consider renaming it to `on_off_invert` to be consistent with the RemoteValueSwitch and Cover attributes.